frontend/src/ofrak/remote_resource.js

```diff
@@ -271,6 +282,8 @@ export class RemoteResource extends Resource {
     ingest_component_results(analyze_results, this.resource_list);
     this.flush_cache();
     this.update();
+
+    await this.update_script();
```

Comment:
The script should update after `queue_patch` below, but doesn't. Please do a pass over the back end routes and make sure that all routes that change the script have a corresponding call to `update_script` in the front end!